ATM.py

```diff
@@ -0,0 +1,141 @@
+__author__ = 'Adi Kadar Levi'
+
+
+class ATM:
+    def __init__(self):
+        self.id_num = 0
+        self.password = 0
+        self.balance = 0
+        self.__pass_dict = {}
+        self.__bal_dict = {}
+
+    def get_passwords_dictionary(self):
+        bank_data_path = r'C:\Users\Elior Mor Yosef\Documents\GitHub' \
+                         r'\python-training\python-training\bank.txt'
+
+        with open(bank_data_path, 'r') as bank_data:
+            for line in bank_data:
+                words = line.split()
+                self.__pass_dict[words[0]] = words[1]
```

Comment:
make the numbers consts

Reply:
and then think of a way for combining the two get functions, they look quite similar.

Reply:
done